Gemfile

```diff
@@ -8,6 +8,13 @@ group :backend, :frontend, :core, :api do
   rails_version = ENV['RAILS_VERSION'] || '~> 6.0.0'
   gem 'rails', rails_version, require: false
 
+  if ENV['CIRCLE_TEST_REPORTS']
```

Comment:
Just curious about `CIRCLE_TEST_REPORTS` var: why not using one of the built-in env flags: https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables ? 🤔(`CI` or `CIRCLECI`, both booleans).

Reply:
I didn't know exactly what env variables were set in circleCi by default, so I went with one that's explicitly set in the file [circle.yml](https://github.com/solidusio/solidus/blob/master/.circleci/config.yml)

After looking at the list I think the best candidate is `CI`, wdyt?

Reply:
Agree 👍 I usually see `ENV['CI']` used in these cases